src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/PlanBulkAlert.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.hmppsalertsapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.entity.BulkPlanRepository
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.entity.getPlan
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.BulkPlan
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.model.request.BulkRequest
+import java.util.UUID
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.entity.BulkPlan as Plan
+
+@Service
+class PlanBulkAlert(private val planRepository: BulkPlanRepository) {
+  fun createNew(): BulkPlan = planRepository.save(Plan()).toModel()
+  fun update(id: UUID, request: BulkRequest): BulkPlan {
+    val plan = planRepository.getPlan(id)
+
+    return plan.toModel()
+  }
```

Comment:
i see that this is a placeholder at the moment?
the UI won't need a response body for this.
anyway, as long as it's not returning the full list of added prisoners in the response, i see no problem in returning a response body.